disnake/message.py

```diff
@@ -2192,13 +2254,76 @@ async def reply(
             reference = self
         return await self.channel.send(content, reference=reference, **kwargs)
 
-    def to_reference(self, *, fail_if_not_exists: bool = True) -> MessageReference:
+    async def forward(
```

Comment:
The `forward` method can also be added to `PartialMessage`, alongside the other `Message.*` methods. Eventually I'd like to rework the hierarchy here to make `Message` inherit from `PartialMessage`, but that's out of scope for now c: